services/wallet/market/market.go

```diff
@@ -133,9 +143,86 @@ func (pm *Manager) FetchTokenMarketValues(symbols []string, currency string) (ma
 	}
 
 	marketValues := result.(map[string]thirdparty.TokenMarketValues)
+	pm.updateMarketCache(currency, marketValues)
 	return marketValues, nil
```

Comment:
Here we're adding a line save the token market values into the `marketCache` when calling `FetchTokenMarketValues` function.